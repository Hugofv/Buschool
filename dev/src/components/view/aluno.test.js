import React from "react";
import Aluno from "./aluno";
import configureStore from "redux-mock-store";
import { shallow, mount } from 'enzyme';
import { Container } from "native-base";

const middlewares = [];
const mockStore = configureStore(middlewares);

const initialState = {
  preferences: {
    save_photos_locally: false,
    open_to_camera: false
  }
};

function setup() {
  const props = {
    fetchAlunos: jest.fn()
  };
  const enzymeWrapper = shallow(
    <Aluno {...props}/>,
    { context: { store: mockStore(initialState) } }
  );

  return {
    props,
    enzymeWrapper
  };
}

describe("<Aluno/>", () => {
    it("Renderizar componentes", () => {
        const { enzymeWrapper } = setup();
        expect(enzymeWrapper.contains(<Container />)).toMatchSnapshot();
    });
});
