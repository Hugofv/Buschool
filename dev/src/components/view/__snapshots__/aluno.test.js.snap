// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aluno/> Renderizar componentes 1`] = `false`;
